modules/migration-running-migration-plan-cam.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// migration/migrating_openshift_3_to_4/migrating-openshift-3-to-4.adoc
+[id='migration-running-migration-plan-cam_{context}']
+= Running a migration plan in the CAM web console
+
+You can migrate applications by running the migration plan you created in the CAM web console.
+
+.Prerequisites
+
+* Connected source cluster(s)
+* Connected target cluster
+* Connected S3 repository
+* Valid migration plan
+
+.Procedure
+
+. Select a migration plan and click *Stage* (copies the data from the source cluster to the target cluster without stopping the application) or *Migrate* (stops the application on the source cluster and moves the data to the target cluster).
+
+. Optionally, if you click *Migrate*, you can select *Do not stop applications on the source cluster during migration*.
+. Click *Migrate*.
```

Comment:
This procedure is a bit confusing. Maybe it would make more sense with the CAM web console in front of me, but are the steps really to either 1). select a plan, click Stage, then click Migrate or to 2). select a plan, click Migrate, then click Migrate again? 

Seems confusing with multiple "Migrate" options. Maybe separate Stage/Migrate from the first step to make them clearer?

Reply:
Not sure how to explain this. "Stage" and "Migrate" are separate steps but they do not require separate procedures, IMO. The differences between the procedures are explained in the "About CAM" section. The user might choose to stage a migration plan more than once, but would only migrate with the migration plan once.

I'll see whether I can make it clearer, without duplicating the procedure.